React hook movie list/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, createContext, useState } from "react";
+
+// context Object which stores some data and will make it available throughout the project
+const ThemeContext = createContext();
+
+const getTheme = () => {
+  const theme = localStorage.getItem("theme");
```

Comment:
can you use session storage or useContext here instead of localstorage

Reply:
sessionstorage is used